gbfs.md

```diff
@@ -388,10 +383,25 @@ Field Name | REQUIRED | Type | Defines
   "version": "3.0",
   "data": {
     "system_id": "example_cityname",
-    "language": "en",
-    "name": "Example Bike Rental",
-    "short_name": "Example Bike",
-    "operator": "Example Sharing, Inc",
+    "languages": ["en"],
```

Comment:
Due to a limitation of GitHub, I can't seem to comment on unmodified text that appears in collapsed sections of the diff, but...

There are still a couple of example `system_information.json` snippets in the file that still reference the old singular "language" field.  Might be good to update them as well?

Reply:
Good catch, I can see some examples still need to be updated.